bears/general/RegexLintBear.py

```diff
@@ -0,0 +1,54 @@
+import re
+
+from queue import Queue
+from sarge import run, Capture
+from contextlib import suppress
+
+from bears.general.AnnotationBear import AnnotationBear
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import execute_bear
+
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+class RegexLintBear(LocalBear):
+    LANGUAGES = {'All'}
+    REQUIREMENTS = {PipRequirement('regexlint', '1.6')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    def run(self, filename, file, language: str):
+        """
+        Bear for linting regex through regexlint.
+
+        :param language:
+            The programming language of the file(s).
+        """
+        section = Section('')
+        section.append(Setting('language', language))
+        bear = AnnotationBear(section, Queue())
+
+        with execute_bear(bear, filename, file) as result:
+            for src_range in result[0].contents['strings']:
+                src_line = src_range.affected_source({filename: file})[0]
+                regex = src_line[src_range.start.column:src_range.end.column-1]
+                with suppress(re.error):
+                    re.compile(regex)
+                    out = run('regexlint --regex "{}"'.format(regex),
+                              stdout=Capture()).stdout.text
+                    if out[-3:-1] != 'OK':
```

Comment:
invert the logic.  `if _some_bad_condition_: continue` , then your yield isnt indented so deeply.  and more conditions will get added in the future.